app/javascript/styles/mastodon/about.scss

```diff
@@ -32,12 +32,39 @@ $fluid-breakpoint: $maximum-width + 20px;
     font-weight: 500;
     counter-increment: list-counter;
 
+    button {
+      background: transparent;
+      border: 0;
+      padding: 0;
+      margin: 0;
+      text-align: start;
+      font: inherit;
+
+      &:hover,
+      &:focus,
+      &:active {
+        background: transparent;
+      }
+
+      &[aria-expanded='false'] .rules-list__hint {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+
+        @supports (-webkit-line-clamp: 2) {
+          display: -webkit-box;
+          -webkit-line-clamp: 2;
+          -webkit-box-orient: vertical;
+          white-space: normal;
+        }
+      }
+    }
+
     &::before {
       content: counter(list-counter);
       position: absolute;
       inset-inline-start: 0;
-      top: 50%;
-      transform: translateY(-50%);
+      top: 1em;
```

Comment:
This introduces some minor spacing issue when there is no hint text:
![image](https://github.com/user-attachments/assets/41e3fdc2-9863-4128-a7cb-267e46bf5657)

Adding `min-height: 3ch` to the `li` elements themselves seems to fix this.